components/ArtGenerator.js

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+export default function ArtGenerator() {
+  return (
+    <section>
+      <Styledh2>Generator #1</Styledh2>
+      <StyledCanvas width="312px" height="312px"></StyledCanvas>
+    </section>
+  );
+}
+
+const StyledCanvas = styled.canvas`
+  border: 1px solid black;
+`;
+
+const Styledh2 = styled.h2`
```

Comment:
```suggestion
const StyledHeadline = styled.h1`
```
This appears to be the main headline of your web page, because there's no other `h1` element, so it might be beneficial to use it. Also, try to avoid setting a naming convention such as `StyledP`, `StyledH1`, `StyledA` ... because you'll run out of options very quickly, and you probably need to style more than just one paragraph per application. ;)